code/process_image.py

```diff
@@ -12,7 +12,7 @@ def process_image(image_path):
     uuid, catalog_number = helpers.extract_qr(image_uri)
     logging.info('Extracted QR codes')
     
-    target_source_path = f'resources/test/sources/source.txt' # f'resources/{institution.lower()}/sources/source.txt'
+    target_source_path = f'resources/{institution.lower()}/sources/source.txt'
```

Comment:
@MichalTorma just fyi, this is how it matches up image bucket folders to dataset folders on the test ipt. `institution` is extracted from the image bucket upload path a few lines up. So you can see it really depends on them doing it in the correct folder structure, it's kind of fragile in that regard.